### DISCUSSION TITLE
Run podman in emulted mode in mac m1

### DISCUSSION THREAD

**Discussion by akumar074** [podman-desktop/podman-desktop#2931] [Q&A]
Hello, I want to know if it's possible to run Podman machine in emulated x86 mode on ARM64 mac. I have followed some tutorials [here](https://developer.ibm.com/tutorials/running-x86-64-containers-mac-silicon-m1/). After making all the changes, it still fails when I run the machine.

**Comment by benoitf** [comment]
hello @akumar074 , with the latest releases of podman, qemu is installed inside the podman machine with x86 mode so there is nothing to do

example, running the httpd amd64 image on a mac intel, you can see there is warning but it's working and if you check the arch it's `x86_64`
```shell
$ podman run --rm -ti --entrypoint=bash docker.io/library/httpd@sha256:760413d133979e7cc5ced07d8c323e7b68bb48b4e114f8898c2f2d6af82e2424                                                                          
WARNING: image platform (linux/amd64) does not match the expected platform (linux/arm64)
root@9c0dc4b81ed5:/usr/local/apache2# arch
x86_64
root@9c0dc4b81ed5:/usr/local/apache2#
```
